packages/editor-ui/src/components/WorkflowLMChat/WorkflowLMChat.vue

```diff
@@ -384,7 +384,12 @@ function extractResponseMessage(responseData?: IDataObject) {
 
 	if (!matchedPath) return JSON.stringify(responseData, null, 2);
 
-	return get(responseData, matchedPath) as string;
+	const matchedOutput = get(responseData, matchedPath);
+	if (typeof matchedOutput === 'object') {
+		return '```json\n' + JSON.stringify(matchedOutput, null, 4) + '\n```';
```

Comment:
Maybe use 2 spaces here? It will be more compact and also in line with the formatting above
```suggestion
		return '```json\n' + JSON.stringify(matchedOutput, null, 2) + '\n```';
```